### PR TITLE
feat: expose dttm at top level of flows stream

### PR DESCRIPTION
**Pull request by nickolasclarke**
This adds a `post_process` method to the FlowsStream class and exposes an `updated` dttm like all other streams that have a dttm like `created` or `updated` passed at the top level.

### CODE REVIEW

File: tap_klaviyo/streams.py [hunk 1]
```diff
@@ -172,6 +172,14 @@ class FlowsStream(KlaviyoStream):
     replication_key = None
```
Comment (edgarrmondragon) [inline tap_klaviyo/streams.py:172]:
```suggestion
    replication_key = "updated"
```

File: tap_klaviyo/streams.py [hunk 2]
```diff
@@ -169,9 +169,17 @@ class FlowsStream(KlaviyoStream):
     name = "flows"
     path = "/flows"
     primary_keys = ["id"]
-    replication_key = None
+    replication_key = "updated"
     schema_filepath = SCHEMAS_DIR / "flows.json"
```
Comment (edgarrmondragon) [inline tap_klaviyo/streams.py:173]: Can we update `flows.json` to include the top-level `updated` field?

Comment (nickolasclarke) [reply tap_klaviyo/streams.py:173]: yes, sorry I was out for a bit. I'll update and push.

Comment (edgarrmondragon) [inline tap_klaviyo/streams.py:173]:
```suggestion
    schema_filepath = SCHEMAS_DIR / "flows.json"
    is_sorted = True
```

**Review by edgarrmondragon — CHANGES_REQUESTED** [review]

**Comment by edgarrmondragon** [comment]
Thanks @nickolasclarke!

**Pull request merged**